src/helpers/Task.js

```diff
@@ -288,7 +289,6 @@ export default class Task {
     name: "Visual Question Answering",
     description: "Used to answer questions based on visual input.",
     id: visualQuestionAnswering,
-
     inputs: [
       {
         inputText: '[Visual Input]',
```

Comment:
@ShivanshShalabh When you have a chance, you should go back and update the `inputText` for a bunch of these tasks to be a bit more explanatory.